disnake/permissions.py

```diff
@@ -1038,6 +1038,14 @@ def send_voice_messages(self) -> int:
         """
         return 1 << 46
 
+    @flag_value
+    def send_polls(self) -> int:
```

Comment:
A couple things i've noticed, unfortunately not as simple since this is one of the permissions that can implicitly be affected by others; generally you can look at similar permissions and check out all the places where they are used, like e.g. `send_voice_messages`:

- `abc.GuildChannel._apply_implict_permissions`, `Thread.permissions_for`, and `Permissions.text()` should handle the new permission too
- running `nox -s codemod -- run-all` will update all the overloads with the new permission and stop CI from complaining